configs/sst_desktop_applications-evolution-data-server.yaml

```diff
@@ -0,0 +1,15 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Evolution Data Server
+  description: Unified backend for programs that work with contacts, tasks, and calendar information
+  maintainer: sst_desktop_applications
+
+  packages:
+  - evolution-data-server
+  - evolution-data-server-langpacks
+  - evolution-ews
+  - evolution-ews-langpacks
```

Comment:
@tpopela please note that evolution-ews includes both e-d-s and evolution plugins, and its continued presence means that evolution itself is still being pulled into ELN.  There does not seem to be a built-in way to build only the e-d-s components.  Furthermore, given your explanation in this PR wrt Exchange support, shouldn't this too be retired, and only e-d-s itself remain?